drivers/serial/uart_mcux_lpuart.c

```diff
@@ -21,6 +21,9 @@
 #include <zephyr/logging/log.h>
 
 #include <fsl_lpuart.h>
+#if CONFIG_NXP_LP_FLEXCOMM
+#include "nxp_lp_flexcomm.h"
```

Comment:
im a bit confused how is this file found ?

Reply:
Updated to move this driver to the `mfd` folder and the include file to `include/zephyr/drivers/mfd`